packages/@aws-cdk/aws-codedeploy/lib/ecs/deployment-config.ts

```diff
@@ -1,23 +1,7 @@
 import { Construct } from 'constructs';
-import { BaseDeploymentConfig, BaseDeploymentConfigOptions, ComputePlatform, IBaseDeploymentConfig } from '../base-deployment-config';
+import { DeploymentConfig, BaseDeploymentConfigOptions, ComputePlatform, IDeploymentConfig } from '../base-deployment-config';
 import { TrafficRouting } from '../traffic-routing-config';
 
-/**
- * The Deployment Configuration of an ECS Deployment Group.
- *
- * If you're managing the Deployment Configuration alongside the rest of your CDK resources,
- * use the {@link EcsDeploymentConfig} class.
- *
- * If you want to reference an already existing deployment configuration,
- * or one defined in a different CDK Stack,
- * use the {@link EcsDeploymentConfig#fromEcsDeploymentConfigName} method.
- *
- * The default, pre-defined Configurations are available as constants on the {@link EcsDeploymentConfig} class
- * (for example, `EcsDeploymentConfig.AllAtOnce`).
- */
-export interface IEcsDeploymentConfig extends IBaseDeploymentConfig {
-}
```

Comment:
This seems like a breaking change compared to an older release like v2.44.0, where IEcsDeploymentConfig was defined as follows:
https://github.com/aws/aws-cdk/blob/v2.44.0/packages/%40aws-cdk/aws-codedeploy/lib/ecs/deployment-config.ts#L13-L16

And fromEcsDeploymentConfigName explicitly returned IEcsDeploymentConfig:
https://github.com/aws/aws-cdk/blob/v2.44.0/packages/%40aws-cdk/aws-codedeploy/lib/ecs/deployment-config.ts#L38-L40

IIUC if someone has Java code using fromEcsDeploymentConfigName in v2.44.0, they will break on updating to this, since the return type is changing for that function and the class is being removed, right?
```java
IEcsDeploymentConfig config = EcsDeploymentConfig.fromEcsDeploymentConfigName(...);
```

Reply:
yeah upon further though we can't change these 😄